projects/dsp-ui/src/lib/action/pipes/array-transformation/reverse.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+    name: 'dspReverse'
+})
+export class ReversePipe implements PipeTransform {
+
+    /**
+     * reverses an array
+     */
+
+    transform(value: any): any {
```

Comment:
I agree, sorting with a pipe seems pretty obvious at first but it seems to cause some major performance issues when scaling. Maybe this can be moved to a service so it can still be used by multiple components in the class?

Reply:
moved to a new service called `sorting.service` in 63afcde and marked the old pipe as deprecated